tests-manual/repro_issues_117.R

```diff
@@ -0,0 +1,190 @@
+library(shiny)
```

Comment:
This example is way too complicated -- I'd hold off adding it until there's a minimal example, and when we understand what the problem actually is. In general, I'd like to minimize the number of additional changes in this pull request.